### DISCUSSION TITLE
Ability to change font and round numbers on tray icons

### DISCUSSION THREAD

**Discussion by hansLindberg** [Rem0o/FanControl.Releases#2548] [Ideas]
Hey, the newest update changed the tray icons, and I'm a grumpy fellow that dislikes change. I find the new font harder to read at a glance, and my cpu icon now uses decimals, making the number smaller and even cutting off part of the number. Therefore I'd like an option to change the font on tray icons, as well as a setting to round numbers, I'm not _that_ picky about a 0,5 degree C difference haha.

![image](https://github.com/Rem0o/FanControl.Releases/assets/10706649/ea179646-965f-416b-90dc-208cf60cc21c)

**Comment by Rem0o** [comment]
Your problem here is the "," number separator I didn't account for. Number should be trimmed. Will fix.

The problem with changing the font is that most font will not fit, or do not fit well within a 16x16 icon.  I would have to add the option also to change the font size and the offset positioning within the icon. The offset is also different for 3 digit vs 2 digit.   It's a total nightmare already for myself to fit 1 font/size.

**Reply by hansLindberg** [reply]
All good! Thanks for taking the time to answer. :D

**Comment by gimme-a-name** [comment]
Hi. I'm a fan and supporter for quite some time. I absolutely love the product :)

In the last update the tray icon stays emtpy, if I select a custom temperature sensor (Mix).
![image](https://github.com/Rem0o/FanControl.Releases/assets/171917507/41020668-5c2a-4682-a9c8-dfa8a8a56bb6)


If I select a normal sensor, the temperature is shown.
![image](https://github.com/Rem0o/FanControl.Releases/assets/171917507/61bac33c-6710-4820-aed9-32d2c0224fb3)

**Comment by gimme-a-name** [comment]
Thanks for the quick fix :)
